code/src/UI/Generation/UserSelection.cs

```diff
@@ -22,6 +22,7 @@ public class UserSelection
         public string ProjectType { get; set; }
         public string Framework { get; set; }
         public string HomeName { get; set; }
+        public string Language { get; set; }
```

Comment:
It makes it easier to compare with other places where it has to be a string and without an extra converter. Also, avoids enum with a '#' in the name.